Feature/homework222

VVELDI opened the pull request:
Реализованы:

Кастомная модель пользователя с email-логином, аватаром, телефоном и страной;

Регистрация и авторизация пользователей;

Ограничение доступа к управлению продуктами;

Отправка приветственного письма;

Механизм модерации продуктов с группой и правами;

Русификация интерфейса и админки;

Валидация форм с цензурой и проверкой цены.

MoDeM13 committed:
Добавил регистрацию новых пользователей

MoDeM13 committed:
Добавил регистрацию новых пользователей

MoDeM13 committed:
Реализовал отправку письма на почту

MoDeM13 committed:
Дополнил env файл

MoDeM13 committed:
Ограничил доступ обычным пользователям. Добавил группу модераторов и команду для создания.

MoDeM13 committed:
Реализованы:

Кастомная модель пользователя с email-логином, аватаром, телефоном и страной;

Регистрация и авторизация пользователей;

Ограничение доступа к управлению продуктами;

Отправка приветственного письма;

Механизм модерации продуктов с группой и правами;

Русификация интерфейса и админки;

Валидация форм с цензурой и проверкой цены.

painassasin commented on aaa.py:
Этот файлик лучше не выкладывать в github. Во-первых, тут хранятся данные для подключения в открытом виде, а этом потенциально опасно - правильнее брать их их настроек проекта. А вообще для подобного используются [кастомные команды](https://habr.com/ru/articles/415049/), чтобы можно было сделать так:
```
python3 manage.py check_email_server
```

MoDeM13 committed:
дален файл с чувствительными данными

MoDeM13 committed:
Удалено поле username

Авторизация по email (уникальное поле USERNAME_FIELD)

Добавлены поля: avatar, phone, country
